src/main/java/core/basesyntax/strategy/operation/ReturnOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void performOperation(FruitTransaction transaction) {
+        int currentQuantity = Storage.storage.get(transaction.getFruitName());
+        Storage.storage.put(transaction.getFruitName(),
+                currentQuantity + transaction.getQuantity());
+    }

```

Comment:
According to the encapsulation principle, direct access to the storage's internal data should be avoided. Consider adding a service layer method to handle the logic of updating the storage.